backend/private-graph/graph/resolver.go

```diff
@@ -3644,6 +3644,34 @@ func (r *Resolver) UpsertDiscordChannel(workspaceId int, name string) (*model.Di
 	}, nil
 }
 
+func normalizeExpressions(column *string, metricTypes []modelInputs.MetricAggregator, expressions []*modelInputs.MetricExpressionInput) ([]*modelInputs.MetricExpressionInput, error) {
```

Comment:
this is responsible for converting the previous multiple `metricTypes` into the new format of metric expressions?

Reply:
yes - it's relevant for anyone using our Grafana integration as it will still be sending requests in the old format. I can make an update to the Grafana integration to use the new format after releasing this, but users may still be on older versions.